tools/README.md

```diff
@@ -0,0 +1 @@
+This directory contains symlinks to binaries for some of our testing tools that are managed by https://github.com/bamarni/composer-bin-plugin
```

Comment:
IIRC you can have the plugin add the executable into your regular vendor/bin so you don't need to maintain these links manually.

Reply:
I saw that feature, but it's usage [is discouraged](https://github.com/bamarni/composer-bin-plugin?tab=readme-ov-file#bin-links-bin-links) so I didn't try it.

Reply:
Ah, got it.

Maybe it makes sense to just use the path from `vendor-bin/bin` directly? It's not that much more complex than the tools one and it seems like less "paperwork" with the tools. :)